packages/core/stories/multiline-input/multiline-input.stories.tsx

```diff
@@ -1,23 +1,20 @@
-import { Button, FlowLayout, Label, Text } from "@salt-ds/core";
-import { MultilineInput } from "@salt-ds/lab";
+import { ChangeEvent, Label, useState } from "react";
```

Comment:
Why are you importing Label from react? :)

Reply:
hehe